website/docs/plugins/ray_launcher.md

```diff
@@ -0,0 +1,184 @@
+---
+id: ray_launcher
+title: Ray Launcher plugin
+sidebar_label: Ray Launcher plugin
+---
+<!-- Add PyPI links etc -->
+
+The Ray Launcher plugin provides 2 launchers: `ray_local` and `ray_aws`. `ray_local` will launch jobs locally on your laptop. `ray_aws` launches jobs remotely on AWS and is built on top of [Ray Autoscaler](https://docs.ray.io/en/latest/autoscaling.html).
+
+Install with
+
+```commandline
+pip install hydra_ray_launcher
+```
+
+<div class="alert alert--info" role="alert">
+NOTE: This plugin depends on Hydra 1.0 which is not yet released, if you want to try it install Hydra from master
+</div><br/>
+
+#### ray_local launcher
+After installation, override `hydra/launcher` in your config or command line:
+
+```yaml
+defaults:
+  - hydra/launcher: ray_local
+```
```

Comment:
the example below is not what you are suggesting that the user do in this line.

Reply:
fixed now.